synapse/lib/layer.py

```diff
@@ -1101,11 +1102,11 @@ async def clone(self, newdirn):
 
                 relname = os.path.join(relpath, name)
 
-                srcpath = s_common.genpath(root, name)
+                srcpath = pathlib.Path(s_common.genpath(root, name))
```

Comment:
why are we using a Path object here? ( and in the cases below )

Reply:
it is the well-accepted normal way to deal with paths in Python. It has built-in methods to do all the things we need. It was already the type of the slab.path.